src/translator_ingest/ingests/signor/signor-ingest.py

```diff
@@ -0,0 +1,175 @@
+import uuid
+import koza
+import pandas as pd
+from typing import Any, Iterable
+
+## the definition of biolink class can be found here: https://github.com/monarch-initiative/biolink-model-pydantic/blob/main/biolink_model_pydantic/model.py
+# * existing biolink category mapping:
+#     * 'Gene': 'biolink:Gene',
+#     * 'Chemical': 'biolink:ChemicalEntity',
+#     * 'Smallmolecule': 'biolink:SmallMolecule',
+#     * 'Phenotype': 'biolink:PhenotypicFeature', -> BiologicalProcess
+#     * 'Protein': 'biolink:Protein',
+# * went through valid check cause there is potential issue:
+#     * 'Antibody': 'biolink:Drug', ## check if all are indeed drug
+#     * 'Complex': 'biolink:MacromolecularComplex',
+#     * 'Mirna': 'biolink:MicroRNA',
+#     * 'Ncrna': 'biolink:Noncoding_RNAProduct',
+
+from biolink_model.datamodel.pydanticmodel_v2 import (
+    Gene,
+    ChemicalEntity,
+    SmallMolecule,
+    PhenotypicFeature,
+    Protein,
+    Drug,
+    MicroRNA,
+    NoncodingRNAProduct,
+    MacromolecularComplexMixin,
+    NamedThing,
+    KnowledgeLevelEnum,
+    AgentTypeEnum,
+    Association,
+    PredicateMapping,
+)
+from translator_ingest.util.biolink import (
+    INFORES_SIGNOR,
+    entity_id,
+    build_association_knowledge_sources
+)
+from koza.model.graphs import KnowledgeGraph
+
+## adding additional needed resources
+BIOLINK_AFFECTS = "biolink:affects"
+BIOLINK_entity_positively_regulated_by_entity = "biolink:entity_positively_regulated_by_entity"
+
+# !!! README First !!!
+#
```

Comment:
not a blocker, but I would remove commented out boiler plate here in the next iteration.